### PR TITLE
Main content

### LINKED ISSUE #21: Main content

### PR DESCRIPTION
**Pull request by kalwabed**
Fix #21

### CODE REVIEW

**Comment by kalwabed** [comment]
Migrate to [headlessui](https://headlessui.com/).

File: src/components/layout/main/section-header.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// TODO: add breadcrumbs
```
Comment (zeetec20) [inline src/components/layout/main/section-header.tsx:1]:
```suggestion
```
remove unused comment

File: src/components/layout/navbar/dropdown.tsx [hunk 2]
```diff
@@ -1,97 +1,33 @@
...
-    <DropdownMenu.Root>
-      <DropdownMenu.Trigger asChild>{trigger}</DropdownMenu.Trigger>
-
-      <DropdownMenu.Portal>
-        <DropdownMenu.Content
-          className="DropdownMenuContent min-w-[220px] rounded-md bg-white p-1 shadow-md"
-          sideOffset={sideOffset}
-          align={align}
+    <Menu as="div" className="relative inline-block text-left">
+      <div>
```
Comment (zeetec20) [inline src/components/layout/navbar/dropdown.tsx:14]:
```suggestion
```
remove unused element

File: src/components/layout/sidebar/nav-link-collapsible.tsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { usePathname } from 'next/navigation'
 import { HiChevronDown } from 'react-icons/hi'
```
Comment (zeetec20) [inline src/components/layout/sidebar/nav-link-collapsible.tsx:2]:
```suggestion
```
remove unused import

File: src/components/layout/main/section-header.tsx (hunk 1 above)
Comment (kalwabed) [reply src/components/layout/main/section-header.tsx:1]: It's important to remind me that we need some breadcrumbs in the near future.

**Review by zeetec20 — APPROVED** [review]
LGTM 🔥🔥🔥

**Pull request merged**